### PR TITLE
GOBMCN2-166 - documentation updates

### PR DESCRIPTION
**Pull request by pythiangoryunov**
user-guide.md
-- add quick references for RAC and DR deployments
-- update of PSU and RU patches

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -109,18 +110,45 @@ Initial steps similar to those of the Single Instance installation.
 
    `./install-oracle.sh --help`
 
-1. Run installation. Only IP address of the first cluster node is required:
+1. Create cluster config file. The file `cluster_config.json` is provided as template
```
Comment (bpj146) [inline docs/user-guide.md:113]: For grammar and brevity, I recommend revising this line to something like:
"Create the cluster configuration file by editing the cluster_config.json file template that is provided with the toolkit."

Comment (bpj146) [inline docs/user-guide.md:113]: Should we tell the user what directory they can find the template in?

File: docs/user-guide.md [hunk 2]
```diff
@@ -109,18 +110,45 @@ Initial steps similar to those of the Single Instance installation.
...
+   and should be updated accordingly.
+
+1. Run installation pointing to the file for planned cluster configuration:
```
Comment (bpj146) [inline docs/user-guide.md:116]: Suggestion:
"Install the database with the path to the cluster configuration file specified on the --cluster-config property:"

File: docs/user-guide.md [hunk 3]
```diff
@@ -109,18 +110,45 @@ Initial steps similar to those of the Single Instance installation.
...
    --backup-dest "+RECO" \
    --ora-swlib-path /u02/swlib/ \
    --ora-swlib-type gcs \
+   --cluster-type RAC \
+   --cluster-config cluster_config.json
+   ```
+
+## Command quick reference for DR deployments
+
+Before creation of standby database the primary host and its database should exist.
```
Comment (bpj146) [inline docs/user-guide.md:130]: Suggestion:
"The primary database must exist before you can create a standby database.

File: docs/user-guide.md [hunk 4]
```diff
@@ -109,18 +110,45 @@ Initial steps similar to those of the Single Instance installation.
...
+
+The primary database creation can be performed using [Single Instance Deployments section](#command-quick-reference-for-single-instance-deployments)<br>
+without or set to NONE `--cluster-type` option.
```
Comment (bpj146) [inline docs/user-guide.md:133]: When you create the primary database, omit the `--cluster-type` option or set it to `NONE`. To create the primary database, see [Single Instance Deployments](#command-quick-reference-for-single-instance-deployments).

File: docs/user-guide.md [hunk 5]
```diff
@@ -109,18 +110,45 @@ Initial steps similar to those of the Single Instance installation.
...
+
+Standby database performed the same way as single instance deployment but with options:<br>
+`--primary-ip-addr` which defines the primary IP address<br>and `--cluster-type DG`.
+
```
Comment (bpj146) [inline docs/user-guide.md:137]: To create a standby database, add the following options to the command options that you used to create the primary database:

+   `--primary-ip-addr ${PRIMARY_IP_ADDR}`
+   `--cluster-type DG`

File: docs/user-guide.md [hunk 6]
```diff
@@ -109,18 +110,45 @@ Initial steps similar to those of the Single Instance installation.
...
+1. Run an installation
```
Comment (bpj146) [inline docs/user-guide.md:138]: Install a standby database:

**Review by bpj146 — COMMENTED** [review]
I suggested several revisions to improve clarity, grammar, and style.

File: docs/user-guide.md (hunk 1 above)
Comment (pythiangoryunov) [reply docs/user-guide.md]: I made the change

Comment (pythiangoryunov) [reply docs/user-guide.md]: I think all other files described do not have any comments about their locations meaning they are in root directory. I would leave it without directory location

File: docs/user-guide.md (hunk 3 above)
Comment (pythiangoryunov) [reply docs/user-guide.md]: I made the change

File: docs/user-guide.md (hunk 5 above)
Comment (pythiangoryunov) [reply docs/user-guide.md:136]: I made the change

File: docs/user-guide.md (hunk 6 above)
Comment (pythiangoryunov) [reply docs/user-guide.md]: I made the change

**Comment by pythiangoryunov** [comment]
I made the changes and pushed them to the branch

**Comment by bpj146** [comment]
There are a couple of outstanding comments in the review still.

Also, my review is only an editorial review. Marc should sign of on the changes from a technical perspective.

File: docs/user-guide.md [hunk 7]
```diff
@@ -109,18 +110,44 @@ Initial steps similar to those of the Single Instance installation.
...
    --ora-swlib-type gcs \
+   --cluster-type RAC \
+   --cluster-config cluster_config.json
+   ```
+
+## Command quick reference for DR deployments
+
+The primary database must exist before you can create a standby database.
+
+When you create the primary database, omit the `--cluster-type` option or set it to `NONE`.<br> To create the primary database, see [Single Instance Deployments section](#command-quick-reference-for-single-instance-deployments).
```
Comment (mfielding) [inline docs/user-guide.md:131]: Please remove HTML <br>

File: docs/user-guide.md [hunk 8]
```diff
@@ -319,6 +346,18 @@ Support")</th>
 <tr>
 <td></td>
 <td>Patch - MOS</td>
+<td>COMBO OF OJVM RU COMPONENT 19.8.0.0.200714 + GI RU 19.8.0.0.200714</td>
```
Comment (mfielding) [inline docs/user-guide.md:349]: What about the previous PSU patches?  Can they be removed now?

**Review by mfielding — COMMENTED** [review]
Looks good; would you mind updating the parameter reference list to include the changes to --primary-ip-addr and cluster-type?

**Review by pythiangoryunov — COMMENTED** [review]
added --primary-ip-addr and option 'DG' to cluster-type description

**Review by pythiangoryunov — COMMENTED** [review]
changes for parameters

**Comment by pythiangoryunov** [comment]
changed information about parameters and format of inventory and log files, added new parameters for cleanup (changed to come through GOBMCN2-171)

**Review by mfielding — APPROVED** [review]

**Pull request merged**